server/profile.py

```diff
@@ -0,0 +1,29 @@
+from app import db
+
+class Profile(db.Model):
+    __tablename__ = 'profile'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    image = db.Column(db.Text)
+    price = db.Column(db.Float(precision=2), nullable=False)
+    ingredients = db.Column(db.String(200), nullable=True)
+    portion = db.Column(db.Integer, nullable=False)
+    required_tools = db.Column(db.String(200), nullable=True)
```

Comment:
I think these are recipe fields. Revisit the mockups and think about the database design in terms of the relationship between users/chefs/recipes and what information is needed in which model.